doc/installation/Developer-Setup.md

```diff
@@ -412,6 +412,28 @@ The email notification service is responsible for saving/deleting email adresses
 
 If you want to start this service or simply see more details regarding this feature you can check out the [README.md](https://github.com/openkfw/TruBudget/blob/master/email-notification/README.md) file.
 
+## Dockerized application
```

Comment:
I think we should add this docu to the top of `## Development Setup` so a developer use this much easier method first.
Also add a sentence in the first lines of the  `## Development Setup` describing that this is the fastest dev setup option.

Reply:
Done